pkg/cloud/azure/actuators/machine_scope.go

```diff
@@ -297,11 +299,19 @@ func updateScope(coreClient controllerclient.Client, credentialsSecret *apicorev
 		return errors.Errorf("Azure resource prefix %v did not contain key %v",
 			secretType.String(), AzureResourcePrefix)
 	}
+	envName := []byte("")
+	envName, ok = secret.Data[AzureEnvironmentKey]
+	if !ok {
+		envName = []byte("AzurePublicCloud")
```

Comment:
Where is this now used as we return?

Reply:
It allows the application to target other azure environments. I am not sure what you are asking here.